contrib/cirrus/setup.sh

```diff
@@ -28,3 +27,18 @@ if [[ "$CIRRUS_TASK_NAME" == "build_cross" ]]; then
 	# TODO: We can move this to quay.io if we reach rate-limits, hopefully that's not gonna happen for netavark
 	echo '  "rustembedded/cross" = "docker.io/rustembedded/cross"'  >> /etc/containers/registries.conf.d/000-shortnames.conf
 fi
+
+if [[ "$CIRRUS_TASK_NAME" == "podman_e2e" ]]; then
+	# build netavark
+	make
```

Comment:
This shouldn't be necessary, the binary should already be in `/var/tmp/netavark/bin/netavark` (because of `bin_cache: *ro_bin_cache`).  If it's not there, something horrible has broken in CI.